packages/graphback-runtime-knex/src/KnexDBDataProvider.ts

```diff
@@ -115,27 +115,27 @@ export class KnexDBDataProvider<Type = any, GraphbackContext = any> implements G
     throw new NoDataError(`No results for ${this.tableName} and id: ${JSON.stringify(ids)}`);
   }
 
-  private usePage(query: Knex.QueryBuilder, page?: GraphbackPage, defaultLimit: number = 10, defaultOffset: number = 0) {
-    if (page) {
-      if (!(page.hasOwnProperty("offset"))) {
```

Comment:
this failed for me giving "page.hasOwnProperty does not exist". It is worth checking if the same issue exists on Mongo provider @wtrocki

Reply:
yep. Page is not required so we just simply need to check for it.
Sorry for introducing this bug. I have separate issue to test pagination.